packages/wallets/src/components/WalletTransferFormAccountSelection/WalletTransferFormAccountSelection.tsx

```diff
@@ -24,7 +25,7 @@ const WalletTransferFormAccountSelection: React.FC<TProps> = ({ label, onSelect
                     className='wallets-transfer-form-account-selection__header__close-button'
                     onClick={() => modal.hide()}
                 >
-                    x
+                    <Close />
```

Comment:
```suggestion
                    <CloseIcon />
```
Best to rename it this